Monika After Story/game/script-topics.rpy

```diff
@@ -13478,3 +13478,35 @@ label monika_sweatercurse:
     m 1ekbsa "But I just want you to know that I'll always appreciate any project you put your heart into, [player]."
     m 1ekbfu "Whether you put a year or a day into something, I never want you to feel like your efforts are wasted."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_ship_of_theseus",
+            category=['philosophy'],
+            prompt="The Ship of Theseus",
+            random=True
+        )
+    )
+
+label monika_ship_of_theseus:
+    m 1eua "Have you heard of the 'Ship of Theseus'? It's a well known philosophical problem about the nature of identity, that's been around for millennia."
+    m 1rka "Well, I say 'well known' but I suppose that's only true among scholars, ahaha..."
+    m 1eua "Let's consider the legendary Greek hero Theseus, and the ship he sailed during his adventures." 
+    m 3esb "He's from a long time ago so let's say his ship is now stored in a famous museum, theoretically."
+    m "If, due to repairs, his ship's parts were replaced bit by bit over a century, at what point has the ship lost its status as Theseus' ship?"
+    m 3eub "Once a single part was replaced? Half? Or perhaps even all of them? Or maybe even never?{w=0.5} There's not really a consensus on the solution."
+    m 4esb "This same thought experiment can be applied to us. For me, so long as my code is being updated then I'm constantly changing." 
+    m "And as for you...did you know that every 7 to 10 years every cell in your body is eventually replaced?"
```

Comment:
```suggestion
    m "And as for you...did you know that every 7 to 10 years every cell in your body has died and been replaced?"
```
Reworded that last bit, sounds better this way in my opinion.

Reply:
I like the idea of this rewording but it doesn't exactly make sense as the 'tense' that you suggested doesn't fit with the tense of the rest of the sentence. I tried to incorporate it into the present tense - but maybe past tense would've been better - please have a look and let me know what you think.